simply-rets-utils.php

```diff
@@ -87,6 +87,17 @@ public static function normalizePropertyType($type) {
         return $normalized_type;
     }
 
+    /**
+     * Remove trailing "County" (or "county") from a county name so it
+     * can be shown within our details builder, or in other use-cases
+     * where "County" may be appended manually.
+     */
+    public static function normalizeCountyText($county) {
+        $county_text = str_replace(" County", "", $county);
```

Comment:
I think we could probably improve this to say: only replace this text *at the end* of the string. But I can't think of any cases where this would be an issue so we'll roll with it and see if anyone has issues. Easy to change.